### PR TITLE
TIED-47: Add Github Actions CI workflow

### PR DESCRIPTION
**Pull request by danipran**
Took a lot of inspiration from https://github.com/City-of-Helsinki/linkedevents/pull/509. Thanks, Matti!

### CODE REVIEW

**Comment by danipran** [comment]
...aaand immediately realized that Elasticsearch needs some more configuring, because of Voikko.

**Review by voneiden — COMMENTED** [review]
Something funky with codecov still.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        run: |
+          python manage.py migrate
+      - name: Check black
+        run: |
+          black --check .
+      - name: Check flake8
+        uses: liskin/gh-problem-matcher-wrap@v2
+        with:
+          linters: flake8
+          run: flake8
```
Comment (tuomas777) [inline .github/workflows/ci.yml:91]: I guess there should be blank lines between these steps as there are blank lines between all the other steps, and there doesn't seem to be any logical reason why these should be grouped together.

**Comment by tuomas777** [comment]
> ...aaand immediately realized that Elasticsearch needs some more configuring, because of Voikko. Edit: Or maybe not. Seems to work fine without any Voikko configurations...?

Yeah, there is probably nothing that tests Voikko, and that is fine at least for now.

**Comment by danipran** [comment]
> Something funky with codecov still.

Yeah, it needs to be configured for this repository, but the Github action itself should work.

**Comment by danipran** [comment]
Well... seems like codecov is working as well. Weird, since I didn't even do anything about it. Oh well. ¯\_(ツ)_/¯

**Comment by voneiden** [comment]
Ok yeah, now it uploaded, but it still shows "Processing" here https://app.codecov.io/github/City-of-Helsinki/helerm/pull/335 and also CI failed for the commit.

Maybe just try merging and keep an eye on it if it starts working?

There are currently four webhooks here https://github.com/City-of-Helsinki/helerm/settings/hooks 

At least two are broken and one is for travis (obsolete).

**Comment by danipran** [comment]
> Maybe just try merging and keep an eye on it if it starts working?

Yup, I agree.

**Review by tuomas777 — APPROVED** [review]
LGTM 👍 🐺

**Pull request merged**